assign_rights/views.py

```diff
@@ -1,42 +1,57 @@
-from django.shortcuts import render
 from django.views.generic import CreateView, DetailView, ListView, UpdateView
+from rest_framework.response import Response
+from rest_framework.views import APIView
 
+from .assemble import RightsAssembler
 
-def RightsShellsListView(ListView):
+
+class RightsShellsListView(ListView):
```

Comment:
This was my error, but if these views are (correctly) being updated from methods to classes, they should also change from plural to singular

Reply:
so `class RightsShellListView` and `GroupingListView` instead of `GroupingsListView`?

Reply:
Nevermind, just checked your branch. Will update to match those.